plugin/plugin.go

```diff
@@ -131,6 +132,42 @@ func (c *GRPCContext) Message() Message {
 func (c *GRPCContext) Emit(data []byte) {
 }
 
+// HasPermission checks whether the sender of the tx has any of the permission given in `roles` on `token`
+func (c *GRPCContext) HasPermission(token []byte, roles []string) (bool, []string) {
+	addr := c.Message().Sender
+	return c.HasPermissionFor(addr, token, roles)
```

Comment:
is this supposed to be here? or did it get mixed in with the roles/permissions PR